beacon_node/network/src/network_beacon_processor/gossip_methods.rs

```diff
@@ -737,6 +735,28 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
                             MessageAcceptance::Ignore,
                         );
                     }
+                    GossipBlobError::PastFinalizedSlot { .. } => {
+                        warn!(
```

Comment:
This log is not actionable by the user, I should not be a warn. We can add a new metric `malicious_peer_activity` to draw attention to us instead

Reply:
This applies to all the warns in this file

Reply:
Kept the warning for the more serious/rare blob verification errors. Like warning for invalid blocks, these are potentially interesting as a canary for network issues

Reply:
If we want to change that behaviour, we can do it in a separate PR. AFAIK the warning that's left has never (?) tripped on mainnet